[2013-12-02 15:52:52] log.ERROR: exception 'PDOException' with message 'SQLSTATE[HY000] [1049] Unknown database 'lib_cit_1.4'' in F:\xampp\htdocs\lib_cit_1.4\vendor\laravel\framework\src\Illuminate\Database\Connectors\Connector.php:47
Stack trace:
#0 F:\xampp\htdocs\lib_cit_1.4\vendor\laravel\framework\src\Illuminate\Database\Connectors\Connector.php(47): PDO->__construct('mysql:host=loca...', 'root', '', Array)
#1 F:\xampp\htdocs\lib_cit_1.4\vendor\laravel\framework\src\Illuminate\Database\Connectors\MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 F:\xampp\htdocs\lib_cit_1.4\vendor\laravel\framework\src\Illuminate\Database\Connectors\ConnectionFactory.php(41): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 F:\xampp\htdocs\lib_cit_1.4\vendor\laravel\framework\src\Illuminate\Database\DatabaseManager.php(128): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 F:\xampp\htdocs\lib_cit_1.4\vendor\laravel\framework\src\Illuminate\Database\DatabaseManager.php(64): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 F:\xampp\htdocs\lib_cit_1.4\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php(2455): Illuminate\Database\DatabaseManager->connection(NULL)
#6 F:\xampp\htdocs\lib_cit_1.4\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php(2421): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 F:\xampp\htdocs\lib_cit_1.4\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php(1510): Illuminate\Database\Eloquent\Model->getConnection()
#8 F:\xampp\htdocs\lib_cit_1.4\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php(1444): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 F:\xampp\htdocs\lib_cit_1.4\vendor\laravel\framework\src\Illuminate\Auth\EloquentUserProvider.php(56): Illuminate\Database\Eloquent\Model->newQuery()
#10 F:\xampp\htdocs\lib_cit_1.4\vendor\laravel\framework\src\Illuminate\Auth\Guard.php(274): Illuminate\Auth\EloquentUserProvider->retrieveByCredentials(Array)
#11 [internal function]: Illuminate\Auth\Guard->attempt(Array)
#12 F:\xampp\htdocs\lib_cit_1.4\vendor\laravel\framework\src\Illuminate\Support\Manager.php(127): call_user_func_array(Array, Array)
#13 F:\xampp\htdocs\lib_cit_1.4\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(206): Illuminate\Support\Manager->__call('attempt', Array)
#14 F:\xampp\htdocs\lib_cit_1.4\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(206): Illuminate\Auth\AuthManager->attempt(Array)
#15 F:\xampp\htdocs\lib_cit_1.4\app\controllers\NguoiDungLogin.php(17): Illuminate\Support\Facades\Facade::__callStatic('attempt', Array)
#16 F:\xampp\htdocs\lib_cit_1.4\app\controllers\NguoiDungLogin.php(17): Illuminate\Support\Facades\Auth::attempt(Array)
#17 [internal function]: NguoiDungLogin->authen()
#18 F:\xampp\htdocs\lib_cit_1.4\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array)
#19 F:\xampp\htdocs\lib_cit_1.4\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('authen', Array)
#20 F:\xampp\htdocs\lib_cit_1.4\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'authen', Array)
#21 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#22 F:\xampp\htdocs\lib_cit_1.4\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#23 F:\xampp\htdocs\lib_cit_1.4\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#24 F:\xampp\htdocs\lib_cit_1.4\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#25 F:\xampp\htdocs\lib_cit_1.4\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#26 F:\xampp\htdocs\lib_cit_1.4\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#27 F:\xampp\htdocs\lib_cit_1.4\index.php(49): Illuminate\Foundation\Application->run()
#28 {main} [] []
[2013-12-02 21:40:11] log.ERROR: exception 'ErrorException' with message 'Undefined offset: 0' in F:\xampp\htdocs\libary_it_1.1\vendor\laravel\framework\src\Illuminate\Support\Collection.php:470
Stack trace:
#0 F:\xampp\htdocs\libary_it_1.1\vendor\laravel\framework\src\Illuminate\Support\Collection.php(470): Illuminate\Exception\Handler->handleError(8, 'Undefined offse...', 'F:\xampp\htdocs...', 470, Array)
#1 F:\xampp\htdocs\libary_it_1.1\app\controllers\General.php(114): Illuminate\Support\Collection->offsetGet(0)
#2 F:\xampp\htdocs\libary_it_1.1\app\storage\views\f996d5fe0fe60a2ce45976bf523b64cc(96): General::getMst(NULL)
#3 F:\xampp\htdocs\libary_it_1.1\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('F:\xampp\htdocs...')
#4 F:\xampp\htdocs\libary_it_1.1\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('F:\xampp\htdocs...', Array)
#5 F:\xampp\htdocs\libary_it_1.1\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('F:\xampp\htdocs...', Array)
#6 F:\xampp\htdocs\libary_it_1.1\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#7 F:\xampp\htdocs\libary_it_1.1\vendor\laravel\framework\src\Illuminate\Http\Response.php(70): Illuminate\View\View->render()
#8 F:\xampp\htdocs\libary_it_1.1\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#9 F:\xampp\htdocs\libary_it_1.1\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#10 F:\xampp\htdocs\libary_it_1.1\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#11 F:\xampp\htdocs\libary_it_1.1\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'index', Object(Illuminate\View\View))
#12 F:\xampp\htdocs\libary_it_1.1\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'index', Array)
#13 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#14 F:\xampp\htdocs\libary_it_1.1\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#15 F:\xampp\htdocs\libary_it_1.1\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#16 F:\xampp\htdocs\libary_it_1.1\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#17 F:\xampp\htdocs\libary_it_1.1\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#18 F:\xampp\htdocs\libary_it_1.1\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#19 F:\xampp\htdocs\libary_it_1.1\index.php(49): Illuminate\Foundation\Application->run()
#20 {main} [] []
